bitcoin/src/blockdata/opcodes.rs

```diff
@@ -351,6 +351,17 @@ pub enum ClassifyContext {
 }
 
 impl Opcode {
+    /// Returns the number of bytes to push if this opcode is a push.
+    // QUESTION: Do we want to return a u8 here instead?
+    pub fn push_bytes(&self) -> Option<u32> {
```

Comment:
In 53bbf143c633c9f92ba8dd27f75f8110d4d2c70c:

I don't think this should be `pub` and I'm skeptical it should be a method at all. Its behavior is pretty weird...it yields a length for the first 76 pushbytes opcodes, but not for PUSHDATA{1,2,4}, nor for any of the numeric pushes.